embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/payload/Event.kt

```diff
@@ -14,15 +14,15 @@ internal data class Event constructor(
 
     @SerializedName("id")
     @JvmField
-    val eventId: String? = null,
+    val eventId: String,
```

Comment:
These are necessary fields that are null-checked - and runtime exceptions are thrown if they are. Instead of doing that, just make them required and non-null.